product_docs/docs/tde/15/secure_key/disabling_key.mdx

```diff
@@ -1,9 +1,9 @@
 ---
 title: Disabling the key wrapping
-description: Learn how skip using a wrapping key.
+description: Learn how to prevent using a wrapping key.
```

Comment:
"Prevent" sounds to me like we are avoiding something bad from happening. But we actually want customers to always use a wrapping key and only skip it for very specific reasons. How about: 
```suggestion
description: Learn how to omit using a wrapping key.
```